src/modules/emote_menu/components/EmoteMenuPopover.module.css

```diff
@@ -19,3 +23,15 @@
     width: 320px;
   }
 }
+
+.resizeHandle {
+  position: absolute;
+  width: 2px;
+  height: 402px;
```

Comment:
could we leverage `bottom: 0; top: 0; left: 0;` with this to avoid specifying an arbitrary height?